From a3eff0ce4b7d04af355e19f08477e1af5f3154c2 Mon Sep 17 00:00:00 2001
From: Mingliang Liu <liuml07@apache.org>
Date: Tue, 13 Dec 2016 10:34:32 -0800
Subject: [PATCH 2115/2848] HADOOP-13900. Remove snapshot version of SDK
 dependency from Azure Data Lake Store File
 System. Contributed by Vishwajeet Dusane

(cherry picked from commit ef34bf2bb92a4e8def6617b185ae72db81450de8)
---
 hadoop-tools/hadoop-azure-datalake/pom.xml |   10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/hadoop-tools/hadoop-azure-datalake/pom.xml b/hadoop-tools/hadoop-azure-datalake/pom.xml
index 8b51a61..8574207 100644
--- a/hadoop-tools/hadoop-azure-datalake/pom.xml
+++ b/hadoop-tools/hadoop-azure-datalake/pom.xml
@@ -35,14 +35,6 @@
     <file.encoding>UTF-8</file.encoding>
     <downloadSources>true</downloadSources>
   </properties>
-  <repositories>
-    <repository>
-      <id>snapshots-repo</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-      <releases><enabled>false</enabled></releases>
-      <snapshots><enabled>true</enabled></snapshots>
-    </repository>
-  </repositories>
   <build>
     <plugins>
       <plugin>
@@ -129,7 +121,7 @@
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>azure-data-lake-store-sdk</artifactId>
-      <version>2.0.4-SNAPSHOT</version>
+      <version>2.0.11</version>
     </dependency>
     <!--  ENDS HERE-->
     <dependency>
-- 
1.7.9.5

